pkg/scheduler/framework/plugins/noderesources/balanced_allocation.go

```diff
@@ -91,7 +91,6 @@ func NewBalancedAllocation(baArgs runtime.Object, h framework.Handle, fts featur
 			scorer:              balancedResourceScorer,
 			useRequested:        true,
 			resourceToWeightMap: resToWeightMap,
```

Comment:
Agree. It may need some small refactoring with `FactoryAdapter`
``
func FactoryAdapter(fts plfeature.Features, withFts PluginFactoryWithFts) PluginFactory {
	return func(plArgs runtime.Object, fh framework.Handle) (framework.Plugin, error) {
		return withFts(plArgs, fh, fts)
	}
}
```

I'd remove it in a following pr.